Taarafo.Portal.Web/Models/Posts/Exceptions/PostDependencyException.cs

```diff
@@ -3,13 +3,14 @@
 // FREE TO USE TO CONNECT THE WORLD
 // ---------------------------------------------------------------
 
+using System;
 using Xeptions;
 
 namespace Taarafo.Portal.Web.Models.Posts.Exceptions
 {
     public class PostDependencyException : Xeption
     {
-        public PostDependencyException(Xeption innerException)
+        public PostDependencyException(Exception innerException)
```

Comment:
Why is it not Xeption?

Reply:
@ShriHumrudha Because i implemented the theory data on DependencyExceptions where the inner Exception is of type Exception.
Please correct me if I'm wrong on this.

![Screenshot 2021-11-20 220102](https://user-images.githubusercontent.com/74690088/142740773-76658d06-5e82-40e7-8c39-b11150934f77.png)

Reply:
var innerException = new Xeption();

Should be the right thing to do.

Reply:
@ShriHumrudha Ok thx for the confirmation. I assume this is because we already on the web level and wrap any exception on the service into failedPostServiceException which is of type Xeption. I assumed that we can not be sure what the service before does and wraps. So thank you for clarification, will change that.